src/specklepy/objects/__init__.py

```diff
@@ -19,5 +11,4 @@
     "units",
     "structural",
```

Comment:
shouldn't structural be removed as well?

Reply:
indeed!